components/forms/input-fields/input-field.scss

```diff
@@ -2,167 +2,152 @@
   display: flex;
   flex-direction: row;
   justify-content: space-around;
-  &__input-container {
+  .msds-input {
     position: relative;
-  }
-
-  &__input-container-input {
-    font-size: 16px;
-    color: $gray-10;
-    height: 56px;
-    width: 100%;
-    padding-left: 20px;
-    border: 1px solid $gray-5;
-    border-radius: 2px;
-    padding-top: 13px;
-
-    &:hover {
-      background-color: $gray-2;
-    }
-
-    &:focus {
-      outline: 4px solid rgba(0, 153, 218, 0.5);
-      border: 1px solid $clear-blue;
-    }
-
-    &:focus ~ label {
-      top: 10px;
-      font-size: 10px;
-    }
-
-    &::placeholder {
-      color: transparent;
-    }
-
-    &:not(:placeholder-shown) ~ label {
-      top: 10px;
-      font-size: 10px;
+    &__icon-wrapper {
+      position: relative;
     }
-
-    &--small {
-      padding-left: 12px;
-      height: 36px;
-      font-size: 12px;
-      line-height: 18px;
-      padding-top: 12px;
-
-      &:focus ~ label {
-        left: 13px;
-        top: 5px;
+    &__text-input {
+      font-size: 16px;
```

Comment:
Instead of worry about styling the font you could import the font using 
    @include msds-font('body-1');

Reply:
ill update the value to variables instead